packages/@sfcc-core/core/.gitignore

```diff
@@ -0,0 +1,4 @@
+*.log
```

Comment:
need .gitignore in each package? Or we can just use our root .gitignore which already has these entries.

Reply:
Tried locally and added this to top level .gitignore file
```suggestion
/packages/@sfcc-core/**/*.d.ts
/packages/@sfcc-core/**/*.js
/packages/@sfcc-core/**/*.js.map

```